Provider.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -1,34 +1,32 @@
 {
-  "object": {
-    "pins": [
-      {
-        "package": "Networking",
-        "repositoryURL": "https://github.com/Lickability/Networking",
-        "state": {
-          "branch": "main",
-          "revision": "93a752f0e1ed5824a57d9bf6e502d14f76a2bc93",
-          "version": null
-        }
-      },
-      {
-        "package": "OHHTTPStubs",
-        "repositoryURL": "https://github.com/AliSoftware/OHHTTPStubs.git",
-        "state": {
-          "branch": null,
-          "revision": "e92b5a5746ef16add2a1424f1fc19529d9a75cde",
-          "version": "9.0.0"
-        }
-      },
-      {
-        "package": "Persister",
-        "repositoryURL": "https://github.com/Lickability/Persister",
-        "state": {
-          "branch": "main",
-          "revision": "ac8f39433b3ced5df28f6c414d06ddc50a57ba7c",
-          "version": null
-        }
+  "pins" : [
+    {
+      "identity" : "networking",
+      "kind" : "remoteSourceControl",
+      "location" : "https://github.com/Lickability/Networking",
+      "state" : {
+        "branch" : "main",
+        "revision" : "0ed53ac918cce33d6273b430da29c296ae58edc2"
       }
-    ]
-  },
-  "version": 1
+    },
+    {
+      "identity" : "ohhttpstubs",
+      "kind" : "remoteSourceControl",
+      "location" : "https://github.com/AliSoftware/OHHTTPStubs.git",
+      "state" : {
+        "revision" : "e92b5a5746ef16add2a1424f1fc19529d9a75cde",
+        "version" : "9.0.0"
+      }
+    },
+    {
+      "identity" : "persister",
+      "kind" : "remoteSourceControl",
+      "location" : "https://github.com/Lickability/Persister",
+      "state" : {
+        "branch" : "main",
+        "revision" : "ac8f39433b3ced5df28f6c414d06ddc50a57ba7c"
+      }
+    }
+  ],
+  "version" : 2
```

Comment:
Interesting, looks like the format changed (version: 2)?

Reply:
Sure did!